.github/workflows/releases.yml

```
@@ -0,0 +1,37 @@
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+' # Triggers the workflow when a tag matching the version pattern is pushed (e.g., 11.1.1)
+
+jobs:
+  generate-release-notes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)